modules/standard/SysError.chpl

```diff
@@ -41,6 +41,19 @@ module SysError {
 
 use SysBasic;
 
+class SystemError : Error {
+  var err: syserr;
+
+  proc SystemError(err: syserr, msg: string) {
+    this.err = err;
+    this.msg = msg;
+  }
+
+  proc writeThis(f) {
+    try! f.write(msg);
```

Comment:
Should writeThis methods be `throws` ? If so we'll need to update buildDefaultFunctions & documentation about readThis/writeThis/readWriteThis. Let's make sure to write down a TODO somewhere.